File: portal_backend/src/entities/work_dailyhours.ts
import {
  Entity,
  PrimaryGeneratedColumn,
  Column,
  ManyToOne,
  JoinColumn
} from 'typeorm'
import { WorkTeamMember } from '../entities/work_team_member'

@Entity({ name: 'work_dailyhours' })
export class WorkDailyHours {
  @PrimaryGeneratedColumn('uuid', { name: 'work_dailyhours_id' })
  workDailyhoursId: string

  @ManyToOne(type => WorkTeamMember)
  @JoinColumn({ name: 'work_team_member_id' })
  workTeamMember: WorkTeamMember

  @Column({ name: 'work_date' })
  workDate: Date

  @Column({ name: 'hours' })
  hours: number
}

File: fg_metrics/VSTS/VSTSComms/VSTSComms/EntitiesMySQL/WorkDailyhours.cs
using System;
using System.Collections.Generic;

namespace VSTSComms.EntitiesMySQL
{
    public partial class WorkDailyhours
    {
        public string WorkDailyhoursId { get; set; }
        public string WorkTeamMemberId { get; set; }
        public int? Hours { get; set; }

        public WorkTeamMember WorkTeamMember { get; set; }
    }
}
